docs/source/operations/projections/airocean.rst

```diff
@@ -0,0 +1,59 @@
+.. _airocean:
+
+********************************************************************************
+Airocean
+********************************************************************************
+
+.. versionadded:: 9.6.0
+
+The Airocean map projection is a icosahedron polyhedral projection that intends
+to lay down continental landmasses into a contiguous map with minimal scale and angular
+distortion while preserving land surfaces from (most) interruptions.
+In comparison to the Snyder's Icosahedral Equal Area, this projection relies on a specific way 
+to orient and cut the icosahedron so that most interruptions lie within the ocean. The projection
+was first invented by Buckminster Fuller in 1943 using a cuboctahedron as basis.
+The idea to use an icosahedron came later in 1954, and it wasn't until 1978 that the projection 
+was formally mathematically defined :cite:`Gray1995`.
+
++---------------------+----------------------------------------------------------+
+| **Classification**  | Polyhedral, equal area                                   |
++---------------------+----------------------------------------------------------+
+| **Available forms** | Forward and inverse, spherical                           |
```

Comment:
The `PROJ_HEAD` has "Sph&Ell" and it appears that ellipsoidal forms are supported, so should the docs reflect this?
```suggestion
| **Available forms** | Forward and inverse, spherical and ellipsoidal           |
```